lib/gems/pending/appliance_console/internal_database_configuration.rb

```diff
@@ -25,6 +25,7 @@ def initialize(hash = {})
 
     def set_defaults
       self.host              = "127.0.0.1"
+      self.port              = default_port
```

Comment:
I think this should not be set here and defaulted in external database configuration ... oops.